components/servo/gpio/servo.go

```diff
@@ -63,15 +63,12 @@ func (config *servoConfig) Validate(path string) ([]string, error) {
 		if config.MaxDeg != nil && *config.StartPos > *config.MaxDeg {
 			return nil, viamutils.NewConfigValidationError(path, errors.New("starting_position_degs cannot be higher than max_angle_deg"))
 		}
-		if *config.StartPos < minDeg || *config.StartPos > maxDeg {
+		if *config.StartPos < defaultMinDeg || *config.StartPos > defaultMaxDeg {
 			return nil, viamutils.NewConfigValidationError(path, errors.New("starting_position_degs should be between 0 and 180"))
 		}
 	}
-	if config.MinDeg != nil && *config.MinDeg < minDeg {
-		return nil, viamutils.NewConfigValidationError(path, errors.Errorf("min_angle_deg cannot be lower than %f", minDeg))
-	}
-	if config.MaxDeg != nil && *config.MaxDeg > maxDeg {
-		return nil, viamutils.NewConfigValidationError(path, errors.Errorf("max_angle_deg cannot be higher than %f", maxDeg))
+	if config.MinDeg != nil && *config.MinDeg < 0 {
```

Comment:
we don't enforce an upper limit for maxDeg since it's dependent on the hardware.  in theory we should also ignore the test minDeg and let a user set a negative minimum deg

Reply:
...so why not do that, too? (i.e., why not get rid of this check and allow a negative minimum value?)